test/physical/swtbahn-standard/testsuite.c

```diff
@@ -125,25 +135,74 @@ void testsuite_case_swtbahnStandardTrackCoverage(const char *train) {
 	testsuite_switch_point("point9", "reverse");
 	testsuite_switch_point("point11", "reverse");
 	
+	sleep(2);
+	// Check that points are in desired position
+	point_check &= testsuite_check_point_aspect("point6", "reverse");
+	point_check &= testsuite_check_point_aspect("point8", "reverse");
+	point_check &= testsuite_check_point_aspect("point2", "reverse");
+	point_check &= testsuite_check_point_aspect("point3", "reverse");
+	point_check &= testsuite_check_point_aspect("point4", "reverse");
+	point_check &= testsuite_check_point_aspect("point5", "reverse");
+	point_check &= testsuite_check_point_aspect("point12", "normal");
+	point_check &= testsuite_check_point_aspect("point10", "reverse");
+	point_check &= testsuite_check_point_aspect("point9", "reverse");
+	point_check &= testsuite_check_point_aspect("point11", "reverse");
```

Comment:
Seems like a helper function should be defined that does the switching and checking of points: it takes a list of points and a corresponding  list of aspects, it calls `testsuite_switch_point` on each pair of point-aspect, it sleeps for `POINT_WAITING_TIME_S` seconds, and then it calls  `testsuite_check_point_aspect` on each pair of point-aspect.

Reply:
Absolutely. I've done something like that when I implemented the physical tests in libbidib-cpp.   
The reason I haven't done it here is that I don't really want to malloc all that stuff; and whilst it is technically possible to do it just statically (as the strings are all known already), I don't remember how to do that correctly when arrays of strings are involved, from a syntax standpoint 😅 .

Reply:
Oh. It's much easier than I expected! Good news.